src/NServiceBus.Core.Tests/Recoverability/SecondLevelRetries/DelayedRetryActionTests.cs

```diff
@@ -0,0 +1,132 @@
+﻿namespace NServiceBus.Core.Tests.Recoverability.SecondLevelRetries
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using DelayedDelivery;
+    using Extensibility;
+    using NServiceBus.Transports;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class DelayedRetryActionTests
+    {
+        const string TimeoutManagerAddress = "timeout handling endpoint";
+        const string EndpointInputQueue = "endpoint input queue";
+
+        [Test]
+        public async Task Should_dispatch_original_message_body()
+        {
+            var dispatcher = new FakeDispatcher();
+            var delayedRetryAction = new DelayedRetryAction(EndpointInputQueue, dispatcher);
```

Comment:
exchanging a one liner with another one liner? (made the dispatcher a field as suggested by you)